other/upgrade_2-1_mysql.sql

```diff
@@ -2631,16 +2631,15 @@ if (in_array('filename', $smileys_columns))
 
 	if (!empty($inserts))
 	{
-		$results = $smcFunc['db_insert']('ignore',
+		$smcFunc['db_insert']('ignore',
 			'{db_prefix}smiley_files',
 			array('id_smiley' => 'int', 'smiley_set' => 'string-48', 'filename' => 'string-48'),
 			$inserts,
 			array('id_smiley', 'smiley_set'),
```

Comment:
You miss the comma in both files.

Reply:
That's fine. PHP is different from SQL in this regard.

Reply:
Which comma?  I'm not seeing it...

Reply:
Oh, the EXTRA comma...   Got it...  Yes, that's a problem & causes the issue below.

Reply:
@Sesquipedalian - This isn't an array, this is a series of parameters in a function call...   If you add that comma, the following ')' is now "unexpected".  I've tested & removing the comma allows the upgrader to proceed.  @albertlast is on target there.

Looks like it will be legal in PHP 7.3...
https://laravel-news.com/php-trailing-commas-functions